src/components/CustomerDetails/InputIdSection.tsx

```diff
@@ -94,9 +92,9 @@ export const InputIdSection: FunctionComponent<InputIdSection> = ({
         </View>
         <SecondaryButton
           text={
-            theme.name === GOVWALLET_THEME_NAME
-              ? i18nt("collectCustomerDetailsScreen", "record")
-              : i18nt("collectCustomerDetailsScreen", "check")
+            c13nt(checkKey) !== checkKey
+              ? c13nt(checkKey)
+              : i18nt("collectCustomerDetailsScreen", checkKey)
```

Comment:
Hmm I'm assuming you're trying to default to `i18nt` when the `c13n` object doesn't contain the translation.

Actually `c13nt`'s 3rd input param takes in a default value. Can see some of the other existing implementations for this. So no need to ownself do ternary check liao

Also how we intended to use c13n was to have the actual string to be displayed as the key and its translation as the value. So something like

```
// c13n/en
{
  "This is a message to be displayed to users": "这是用户会看到的讯息"
}
```

Not sure your implementation follows this pattern?

Reply:
Discussed with Dalson on Slack. Here's the summary

* Dalson's idea was that for the same location of any copy, both govwallet and sally themed campaigns should use the same key in c13n
* It makes a lot of sense to use the same key. But perhaps the key can still default to the copy for sally themed campaigns^, so that the defaulting to key in `c13nt` function can continue to do its job
* Then in this case we also don't have random key strings lying around liao

What do you think?

^ Example
```
// for sally themed
{
  "Check": "Check"
}


// for govwallet themed
{
  "Check": "Record"
}
```

Reply:
thanks for the summary! I've made the change to adhere to the example above, with the GovWallet theme stored in the SSM c13n params (`en` & `zh`)!